gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -169,12 +171,25 @@ def _merge_headers(known_args, pipeline_args, pipeline_mode):
     else:
       headers |= vcf_header_io.ReadVcfHeaders(known_args.input_pattern)
 
-    _ = (headers
-         | 'MergeHeaders' >> merge_headers.MergeHeaders(
-             known_args.split_alternate_allele_info_fields)
-         | 'WriteHeaders' >> vcf_header_io.WriteVcfHeaders(
-             known_args.representative_header_file))
-
+    merged_headers = (headers
```

Comment:
Consider renaming to `merged_header` since we expect one header object at this point.

Reply:
Done